apps/cli/src/commands/doctor.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from "@oclif/core";
+import { execa } from "execa";
+import semver from "semver";
+
+export default class DoctorCommand extends Command {
+    static description =
+        "Verify the minimal requirements for the sunodo execution commands";
+
+    static examples = ["<%= config.bin %> <%= command.id %>"];
+
+    private static MINIMUM_DOCKER_VERSION = "1.13.0"; // Replace with our minimum required Docker version
```

Comment:
Let's set the bar high and set this to `23.0.0`

Reply:
Fixed !